bftengine/include/bftengine/KeyExchangeManager.hpp

```diff
@@ -18,7 +18,7 @@
 #include "Metrics.hpp"
 #include "secrets_manager_impl.h"
 #include "SysConsts.hpp"
-#include "crypto_utils.hpp"
```

Comment:
KeyExchange manager should be using EdDSA after this change. Do we need to include cryptopp_utils.hpp here?